app/src/main/java/com/github/sdp_begreen/begreen/activities/MainActivity.kt

```diff
@@ -222,6 +227,28 @@ class MainActivity : AppCompatActivity() {
             R.id.mainNavDrawSettings -> {
                 replaceFragInMainContainer(SettingsFragment())
             }
+            // handle the "Logout" button in the navigation drawer of the app responsible for
+            // logging out a user who has signed in with Google Sign-In
+            R.id.mainNavDrawLogout -> {
+                val gso = GoogleSignInOptions.Builder(GoogleSignInOptions.DEFAULT_SIGN_IN)
+                    .requestIdToken(getString(R.string.default_web_client_id))
+                    .requestEmail()
+                    .build()
+
+                mGoogleSignInClient = GoogleSignIn.getClient(this, gso)
+
+                mGoogleSignInClient.signOut().addOnCompleteListener {
+                    val intent = Intent(this, SignInActivity::class.java)
+
+                    // short toast message to the user indicating that they are being logged out
+                    Toast.makeText(this, "Logging Out", Toast.LENGTH_SHORT).show()
```

Comment:
This should be in string resources since it would need to be localized (e.g., if you made the app available in both English and French)